src/examples/ParkingLot/Input/paraCfgCurrent.txt

```diff
@@ -0,0 +1,17 @@
+ParameterName	ParameterValue
+environmentFileName	ParkingLot.xml
+generalizedScenario	1
+indoorSwitch	0
+mobilitySwitch	0
+mobilityType	2
+numberOfNodes	2
+numberOfTimeDivisions	10
+referencePoint	[3,3,2]
```

Comment:
You can consider any other referencePoint. Right now, I am considering selectPlanesByDist	0
so referencePoint value does not matter. I will share the updated Parking Lot scenario with Cars once I will be done with Camillo.

Reply:
yes, i agree that setting `selectPlanesByDist 0` the referencePoint is ignored, but since this is part of the examples I'd rather keep a clean and meaningful code. If these is no reason to set it to that specific location, then please set it to [0,0,0]

Reply:
Done. I have inlcuded selectPlanesByDist = 0 with [0 0 0]